puzzles/vault_puzzles/restriction_puzzles/wrappers/recovery.clsp

```diff
@@ -0,0 +1,94 @@
+; This is a highly specific puzzle that enforces that any new create coins are a 1of2 w/ a fixed left side
+; and a variable right side that is put under pre-committed restrictions.
+;
+; The framework this is designed to fit in doesn't really support subtree behavior like this, so the specificity and
+; limited flexibility of this puzzle is due to the fact that it's sort of a hack to support an existing feature with
+; a framework poorly designed for it.
+(mod
+  (
+    DELEGATED_PUZZLE_FEEDER_MOD_HASH
+    1_OF_N_MOD_HASH
+    LEFT_SIDE_SUBTREE_HASH_HASH
+    INDEX_WRAPPER_HASH
+    NONCE
+    RESTRICTION_MOD_HASH
+    MEMBER_VALIDATOR_LIST_HASH
+    DPUZ_VALIDATOR_LIST_HASH
+    Conditions
+    new_right_side_member_hash
+  )
+
+  (include curry.clib)
+  (include utility_macros.clib)
+  (include condition_codes.clib)
+  (defconstant ONE 1)
+
+  (defun check_coins (conditions fixed_puzzle_hash)
+    (if conditions
+      (if (any
+          (= (f (f conditions)) SEND_MESSAGE)
+          (= (f (f conditions)) CREATE_PUZZLE_ANNOUNCEMENT)
+          (= (f (f conditions)) CREATE_COIN_ANNOUNCEMENT)
```

Comment:
Rather than making two separate puzzles, I would prefer that this additional restrictiveness is implemented as a new restriction on the delegated puzzle like `disallow_side_effects.clsp`.

Reply:
I split this up into separate restrictions and removed the clone of force_1_of_2